spec/unit/util/execution_spec.rb

```diff
@@ -549,38 +607,83 @@ def stub_process_wait(exitstatus)
         Puppet::Util::Execution.execute('test command', {:squelch => true, :combine => false})
       end
 
-      it "should read and return the output if squelch is false" do
-        stdout = Puppet::FileSystem::Uniquefile.new('test')
-        Puppet::FileSystem::Uniquefile.stubs(:new).returns(stdout)
-        stdout.write("My expected command output")
+      describe "on POSIX", :if => Puppet.features.posix? do
+        it "should read and return the output with external encoding if squelch is false" do
+          r, w = IO.pipe
+          IO.expects(:pipe).returns([r, w])
+          w.write("My expected ❄ command output")
```

Comment:
to prevent possible errors in display/parse, would you mind updating to use the "\u" version of the snowflake character, ie "\u2744" ?  we've generally been trying to do that consistently as a part of all the utf-8 / i18n work